sdk/python/Pipfile

```diff
@@ -7,6 +7,7 @@ name = "pypi"
 protobuf = ">=3.6.0"
 grpcio = ">=1.9.1"
 dill = ">=0.3.0"
+six = ">=1.12.0"
```

Comment:
do we strictly need `six`?  I would have thought that was only needed for Python2 compatibility?

Reply:
I think you're correct. I used it to match `protobuf`, so I'd like to keep it as-is if that's okay.